components/properties/src/maps.rs

```diff
@@ -216,3 +216,204 @@ where
 {
     get_cp_map(provider, key::SENTENCE_BREAK_V1)
 }
+
+//
+// General_Category Predicate Functions
+// 
+//   For querying a specific code point for inclusion in the union set of more
+//   than one category (General_Category property value).
+//
+
+
+
+
+/// Return whether the code point belongs in the multi-value category `Cased_Letter`.
+/// This is defined as the union of `GeneralSubcategory::UppercaseLetter`,
+/// `GeneralSubcategory::LowercaseLetter`, and `GeneralSubcategory::TitlecaseLetter`.
+/// 
+/// ```
+/// use icu::properties::{maps, GeneralSubcategory, GeneralCategory};
+/// use icu_codepointtrie::CodePointTrie;
+///
+/// let provider = icu_testdata::get_provider();
+/// let payload =
+///     maps::get_general_category(&provider)
+///         .expect("The data should be valid");
+/// let data_struct = payload.get();
+/// let gc = &data_struct.code_point_trie;
+/// 
+/// assert_eq!(gc.get('A' as u32), GeneralSubcategory::UppercaseLetter);
+/// assert!(maps::is_cased_letter(&gc, 'A' as u32));
+/// ```
+pub fn is_cased_letter(gc_map: &CodePointTrie<GeneralSubcategory>, code_point: u32) -> bool {
+    let gc_val = gc_map.get(code_point);
+    0 != (gc_val as u32) & GeneralCategory::CasedLetter.0
+}
```

Comment:
Two other ways:

```rust
impl GeneralSubcategory {
    pub fn is_contained(gc: GeneralCategory) -> bool {
        // do the test
    }
}
```

```rust
impl GeneralCategory {
    pub fn contains(gsc: GeneralSubcategory) -> bool {
        // opposite of above
    }
}
```

Reply:
Done before a previous review. Took the second approach.